orga/app/components/routes/authenticated/sco-students/list-items.js

```diff
@@ -31,4 +35,24 @@ export default class ListItems extends Component {
   closeDissociateModal() {
     this.set('isShowingDissociateModal', false);
   }
+
+  @action
+  async generateUsernameWithTemporaryPassword(student) {
+    const schoolingRegistrationDependentUser = this.store.createRecord('schooling-registration-dependent-user', {
+      organizationId: this.currentUser.organization.id,
+      schoolingRegistrationId: student.id
+    });
+
+    try {
+      await schoolingRegistrationDependentUser.save({ adapterOptions: { generateUsernameAndTemporaryPassword: true } });
+      this.username = schoolingRegistrationDependentUser.username;
+      this.generatedPassword = schoolingRegistrationDependentUser.generatedPassword;
+      this.refreshModel();
+    } catch (response) {
+      const error = _.get(response, 'errors[0]');
+      const detail = _.get(error, 'detail', 'Une erreur est survenue, veuillez réessayer ultérieurement.');
```

Comment:
On devrais pouvoir simplifier à ca:

```suggestion
      const errorDetail = _(response)
           .get('errors[0]')
           .get('detail', 'Une erreur est survenue, veuillez réessayer ultérieurement.');
```

Reply:
Merci , done